templates/README.md

```diff
@@ -153,6 +153,63 @@ Distribution of substantively modified versions of this document is prohibited w
 
 Distribution of the work or derivative of the work in any standard (paper) book form for commercial purposes is prohibited unless prior permission is obtained from the copyright holder.
 {{ end }}
+
+{{ if eq (ds "config").license "MIT" }}
+## License 
+The MIT License (MIT)
+
+Copyright (c) <2018> <Cloud Posse, LLC>
```

Comment:
The `<` and `>` characters should only surround links in markdown. Remove them from here.

Reply:
Use `2017-{{ time.Now.Year }}` everywhere we reference a year

Reply:
Remove the `Copyright` as it's redundant with the `## Copyright` section.